tests/core_tests.py

```diff
@@ -719,26 +721,11 @@ def test_mssql_engine_spec_pymssql(self):
             (1, 1, datetime.datetime(2017, 10, 19, 23, 39, 16, 660000)),
             (2, 2, datetime.datetime(2018, 10, 19, 23, 39, 16, 660000)),
         ]
-        df = dataframe.SupersetDataFrame(
+        table = SupersetTable(
             list(data), [["col1"], ["col2"], ["col3"]], MssqlEngineSpec
         )
-        data = df.data
-        self.assertEqual(len(data), 2)
-        self.assertEqual(
-            data[0],
-            {"col1": 1, "col2": 1, "col3": pd.Timestamp("2017-10-19 23:39:16.660000")},
-        )
-
-    def test_mssql_engine_spec_odbc(self):
```

Comment:
Was thinking when I ripped this out that it was no longer necessary due to transformations [here](https://github.com/apache/incubator-superset/blob/master/superset/db_engine_specs/mssql.py#L64), but it looks like additional transformation is required. cc @chinhngt

Reply:
Refactored and added some additional tests around this. Should be good to go.